examples/darwin-framework-tool/BUILD.gn

```
@@ -280,6 +284,10 @@ executable("darwin-framework-tool") {
     defines += [ "MTR_ENABLE_PROVISIONAL=1" ]
   }
 
+  if (is_leaks) {
+    defines += [ "DFT_ENABLE_LEAKS=1" ]
```

```suggestion
    defines += [ "DFT_ENABLE_LEAK_CHECKING=1" ]
```
Or maybe DFT_ENABLE_LEAK_CHECK?  Because "ENABLE_LEAKS" sounds like "turn on leaking".